### DISCUSSION TITLE
Ai tags refresh bottom - bulk refresh for all or a list

### DISCUSSION THREAD

**Discussion by feickoo** [karakeep-app/karakeep#422] [Ideas]
First of all, thank you for this amazing app. I really love it. 
It would be nice to refresh ai tags with different model to test with. Now I can only refresh ai tag one by one.

bulk refresh ai tags with different models/languages. Is it possible to tag in 2 or 3 languages?

**Comment by kamtschatka** [comment]
you can retrigger inference for all bookmarks in the admin panel

**Reply by feickoo** [reply]
Thanks, this works.
